third_party/2and3/requests/utils.pyi

```diff
@@ -31,7 +31,8 @@ def get_encoding_from_headers(headers): ...
 def stream_decode_response_unicode(iterator, r): ...
 def iter_slices(string, slice_length): ...
 def get_unicode_from_response(r): ...
-
+def quote(string, safe=..., encoding=..., errors=...)
```

Comment:
This is syntactically incorrect; you need `: ...` as the body.

Also, would you mind adding types to the parameters and output? I'm guessing most of them are `str`, but maybe not all, and in Python 2 it may support both bytes and Text.

Reply:
```suggestion
def quote(string: str, safe: str=..., encoding: Union[str, bytes]=..., errors: Union[str, bytes]=...): ...
```

I'm not sure how they are treated in different versions of python, but in python 3.7 `string` and `safe` may be either str or bytes.  Here is the full doc signature and docstring ... 
```
In [3]: requests.utils.quote?
Signature: requests.utils.quote(string, safe='/', encoding=None, errors=None)
Docstring:
quote('abc def') -> 'abc%20def'

Each part of a URL, e.g. the path info, the query, etc., has a
different set of reserved characters that must be quoted.

RFC 3986 Uniform Resource Identifiers (URI): Generic Syntax lists
the following reserved characters.

reserved    = ";" | "/" | "?" | ":" | "@" | "&" | "=" | "+" |
              "$" | "," | "~"

Each of these characters is reserved in some component of a URL,
but not necessarily in all of them.

Python 3.7 updates from using RFC 2396 to RFC 3986 to quote URL strings.
Now, "~" is included in the set of reserved characters.

By default, the quote function is intended for quoting the path
section of a URL.  Thus, it will not encode '/'.  This character
is reserved, but in typical usage the quote function is being
called on a path where the existing slash characters are used as
reserved characters.

string and safe may be either str or bytes objects. encoding and errors
must not be specified if string is a bytes object.

The optional encoding and errors parameters specify how to deal with
non-ASCII characters, as accepted by the str.encode method.
By default, encoding='utf-8' (characters are encoded with UTF-8), and
errors='strict' (unsupported characters raise a UnicodeEncodeError).
File:      ~/anaconda3/lib/python3.7/urllib/parse.py
Type:      function
```